actions/subscribers.go

```diff
@@ -104,6 +105,26 @@ func PostSubscriber(c *gin.Context) {
 		c.JSON(http.StatusBadRequest, err)
 		return
 	}
+	user := middleware.GetUser(c)
+
+	boundariesvc := boundaries.New(storage.GetFromContext(c))
+
+	limitexceeded, _, err := boundariesvc.SubscribersLimitExceeded(user)
+	if err != nil {
+		logger.From(c).WithError(err).WithField("user_id", user.ID).Error("Unable to check subscribers limit for user.")
```

Comment:
user_id is already included in the logger via a request middleware, we don't need to include it in our actions @gudgl @Dzalevski